In [1]:
%%bash
whoami

victor


In [1]:
%%bash
date

Thu Oct 15 16:45:03 CDT 2020


CDT significa Hora de Verano Central (Central Daylight Time)

In [2]:
%%bash
cal 2 1911

   February 1911      
Su Mo Tu We Th Fr Sa  
          1  2  3  4  
 5  6  7  8  9 10 11  
12 13 14 15 16 17 18  
19 20 21 22 23 24 25  
26 27 28              
                      


Su dia de nacimiento cayo en sabado

In [4]:
%%bash
cal 7 1913

     July 1913        
Su Mo Tu We Th Fr Sa  
       1  2  3  4  5  
 6  7  8  9 10 11 12  
13 14 15 16 17 18 19  
20 21 22 23 24 25 26  
27 28 29 30 31        
                      


In [5]:
%%bash
cal -j 7 1913

         July 1913           
 Su  Mo  Tu  We  Th  Fr  Sa  
        182 183 184 185 186  
187 188 189 190 191 192 193  
194 195 196 197 198 199 200  
201 202 203 204 205 206 207  
208 209 210 211 212          
                             


Alberto Barajas nacio el dia 198 en jueves del año de 1913

In [9]:
%%bash
uname -a

Linux DESKTOP-96AH5UB 4.4.0-18362-Microsoft #1049-Microsoft Thu Aug 14 12:01:00 PST 2020 x86_64 x86_64 x86_64 GNU/Linux


La version de Kernel es 4.4.0-18362-Microsoft

PARTE 2 DE LA PRACTICA

In [1]:
%%bash
pwd

/mnt/c/Users/HP/Documents


In [2]:
%%bash
mkdir HumbertoMoya

In [3]:
%%bash
mkdir HumbertoMoya/Hernandez

In [4]:
%%bash
mkdir HumbertoMoya/Moya

In [31]:
%%bash
touch .prueba1

In [37]:
%%bash
touch prueba2

In [33]:
%%bash
mv .prueba1 HumbertoMoya/Hernandez

In [38]:
%%bash
mv prueba2 HumbertoMoya/Moya

El comando cd- te ayuda a regresar al directorio anterior

In [39]:
%%bash
ls -R HumbertoMoya

HumbertoMoya:
Hernandez
Moya

HumbertoMoya/Hernandez:

HumbertoMoya/Moya:
prueba2


No aparecio el archivo con punto inicial

In [41]:
%%bash
ls -Ra HumbertoMoya

HumbertoMoya:
.
..
Hernandez
Moya

HumbertoMoya/Hernandez:
.
..
.prueba1

HumbertoMoya/Moya:
.
..
prueba2


Poniendo el comando ls junto con las opciones -R y la opcion -a, nos lista el contenido de directorios y subdirectorios,
ademas de mostrar todo el contenido del directorio, incluyendo los archivos y directorios ocultos.